tests/e2e/test_pvc_creation_performance.py

```diff
@@ -3,8 +3,10 @@
 """
 import logging
 import pytest
+import math
 import ocs_ci.ocs.exceptions as ex
-
+import ocs_ci.ocs.resources.pvc as pvc
+from concurrent.futures import ThreadPoolExecutor
```

Comment:
IMO, each interface (FS/RBD) should be reported to a different polation id

Reply:
can we consider this not as a blocker for this PR?
I can create a followup PR with Polarion ID for different pools

Reply:
yes :)